var/spack/repos/builtin/packages/process-in-process/package.py

```diff
@@ -9,14 +9,24 @@
 class ProcessInProcess(Package):
     """Process-in-Process"""
 
-    homepage = "https://github.com/RIKEN-SysSoft/PiP"
-    git = "https://github.com/RIKEN-SysSoft/PiP.git"
+    enable_pip_v3 = False
+    enable_pip_gdb = False
+    # we cannot install PiP-gdb (PiP-aware GDB)
+    # since the ncureses package cannot be installed by using Spack
+    # --> https://github.com/spack/spack/issues/8675
+
+    homepage = "https://github.com/procinproc/procinproc.github.io"
+    git = "https://github.com/procinproc/PiP.git"
 
     maintainers = ["ahori"]
 
+    conflicts("platform=darwin", msg="Darwin is not supported.")
+    conflicts("platform=windows", msg="Windows is not supported.")
+
     # PiP version 1 is obsolete
     version("2", branch="pip-2", preferred=True)
-    version("3", branch="pip-3")  # experimental
+    if enable_pip_v3:
```

Comment:
Can we have this unconditionally? Also, can we use a commit or a release rather than a branch?

Reply:
No, we cannot do that. This might be my fault but the PiP-v2 and PiP-v3 are quite different.